### ISSUE TITLE
Scan also for unused addresses

### ISSUE THREAD

**Issue by jcfrane** [LedgerHQ/xpub-scan#109]
Can we have an option to include unused addresses (those without balance) to appear on output? That would be a great help if we experience gap limit error.

**Comment by glethuillier** [comment]
By default, the gap limit is 20 addresses. It can be extended by changing the value of [this variable](https://github.com/LedgerHQ/xpub-scan/blob/4fe252136f87e53dd7ff169aded4e85d7f14ef99/src/configuration/settings.ts#L27) (_e.g._, `const DEFAULT_GAP_LIMIT = 1000`). 

Therefore, if you experience a gap limit error, it will be detected in the range that you provide through this variable, and the used addresses will be displayed like this:

```
SegWit          m/1/0          {active address 1, pre gap}   0               +0.30998771     ←	-0.30998771     →
SegWit          m/1/1          {active address 2, pre gap}   0               +0.29764036     ←	-0.29764036     →
SegWit          m/1/1001       {active address 3, post gap}  0               +0.09834688     ←	-0.09834688     →
SegWit          m/1/1002       {active address 4, post gap}  0.01178649      +0.01178649     ←	-0              →
SegWit          m/1/1003       {active address 5, post gap}  0.02907135      +0.02907135     ←	-0              →
...
```

In other words, the unused addresses belonging to the gap are never displayed, but the used addresses outside the gap are. The gap is rendered implicitly: it can be inferred from the derivation paths (in this example: `m/1/1` ⇨ `m/1/1001`).

Do you think that this is helpful, or would you still prefer to have a way to display the unused addresses belonging to the gap?
